fe/fe-core/src/main/java/com/starrocks/persist/DictionaryMgrInfo.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.persist;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+public class DictionaryMgrInfo implements Writable {
+    @SerializedName(value = "nextTxnId")
+    private long nextTxnId = 1L;
+    @SerializedName(value = "lastSuccessTxnId")
+    private long lastSuccessTxnId = -1L;
+    @SerializedName(value = "nextDictionaryId")
+    private long nextDictionaryId = 1L;
+
```

Comment:
Same to Dictionary.java, should add empty constructor

Reply:
done